packages/aws-cdk-lib/aws-appconfig/lib/environment.ts

```diff
@@ -561,15 +561,17 @@ export interface IEnvironment extends IResource {
   addExtension(extension: IExtension): void;
 
   /**
-   * Grant the given identity the specified actions
-   * @param grantee the identity to be granted the actions
-   * @param actions the data-access actions
+   * Adds an IAM policy statement associated with this environment to an IAM principal's policy.
+   *
+   * @param grantee the principal (no-op if undefined)
+   * @param actions the set of actions to allow (i.e., 'appconfig:GetLatestConfiguration', 'appconfig:StartConfigurationSession', etc.)
    */
   grant(grantee: iam.IGrantable, ...actions: string[]): iam.Grant;
 
   /**
-   * Grants read configuration permissions for this environment
-   * to an IAM principal (Role/Group/User).
+   * Permits an IAM principal a configuration read operations on this environment.
```

Comment:
I think how you wrote it before `Grants ...` was already good. My comment was rather a question if the term `IAM principal` is correct here? I'm not sure if an IAM principal is the same scope like an Grantable. I was expecting a comment like `Grants a resource read permission on the configurations of this environment`
What do you think? I would be happy to hear your opinion here :)